unity-renderer/Assets/Scripts/MainScripts/DCL/Analytics/SupportAnalytics/SupportAnalytics.cs

```diff
@@ -0,0 +1,28 @@
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace SupportAnalytics
+{
+    public class SupportAnalytics : ISupportAnalytics
+    {
+        private const string PLAYER_OPENED_SUPPORT = "player_opened_support";
```

Comment:
is there a more descriptive name than `PLAYER_OPENED_SUPPORT`? maybe `eventName`?

Reply:
the action is the player opening the support panel, it follows the naming we have in other analyitcs as well